### ISSUE TITLE
discovery-dns doesn't perform PTR lookup

### ISSUE THREAD

**Issue by DavieDavieDave** [nextdns/nextdns#287]
I've enabled the `report-client-info` option and `discovery-dns` pointed at my authoritative DNS server (bind9) which is correctly setup to resolve PTR records. However, the nextdns client doesn't attempt to resolve the requesting host name from the authoritative server. I ran a `tcpdump` to see if any PTR requests were being made, however there were none. I am running bind on a non-standard port so that nextdns and bind can co-exist on the same machine. All I see in the logs and analytics within my.nextdns.io, I see `Device #AB123`.

```
control /var/run/nextdns.sock
log-queries false
report-client-info true
use-hosts false
setup-router false
bogus-priv false
listen :53
config [removed]
cache-max-age 0s
max-ttl 0s
hardened-privacy false
timeout 5s
forwarder lan.=127.0.0.1:5353
forwarder lab.=127.0.0.1:5353
forwarder 16.172.in-addr.arpa.=127.0.0.1:5353
discovery-dns=127.0.0.1:5353
cache-size 10M
detect-captive-portals false
auto-activate false
```
```
# nextdns discovered
{
    "dns": {
        "172.16.11.50": []
    },
    "hosts": {
        "localhost": [
            "127.0.0.1",
            "::1"
        ],
        "localhost.": [
            "::1",
            "127.0.0.1"
        ],
        "localhost.localdomain.": [
            "127.0.0.1",
            "::1"
        ],
        "localhost.my.domain.": [
            "::1",
            "127.0.0.1"
        ]
    }
}
```
```
# host 172.16.11.50
50.11.16.172.in-addr.arpa domain name pointer dave-pc.lan.
```
```
# tcpdump -ilo0 udp port 5353 | grep PTR
tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
listening on lo0, link-type NULL (BSD loopback), capture size 262144 bytes
^C16 packets captured
194 packets received by filter
0 packets dropped by kernel
```

**Context**
 - Version [e.g. 1.5.2]:  1.7.0
 - Platform [e.g. macOS, ASUS-Merlin]: FreeBSD

**Comment by ralban** [comment]
I think I had this symptom too. I had configured "in-addr.arpa." forwarder entries in my config to direct local requests for LAN reverse lookups to my DHCP servers. Requests were not resolved; bogus-priv was set to false.

**Comment by rs** [comment]
Fixed by 7106d9b4a265f93e32ff1b842e87e90cea7e480e
